custom_components/iotawatt/sensor.py

```diff
@@ -79,23 +107,28 @@ def device_state_attributes(self):
         else:
             channel = "N/A"
 
-        return {"Type": self._io_type, "Channel": channel}
+        attrs = {
+                "type": self._io_type,
+                "channel": channel
+            }
 
-    @property
-    def unit_of_measurement(self):
-        """Return the unit of measurement of this entity, if any."""
-        unit = self.coordinator.data["sensors"][self._ent].getUnit()
-        if unit == "Watts":
-            return POWER_WATT
-        if unit == "Volts":
-            return VOLT
-        return self.coordinator.data["sensors"][self._ent].getUnit()
+        if last_reset := self.last_reset:
+            attrs["last_reset"] = last_reset.isoformat()
```

Comment:
You already define a last reset property ?

Reply:
I think that was one of my desperate attempts to make statistics work :sweat_smile: I'll remove it.

FWIW, in the end statistics were not working because `IotaWattEntity` entity did not inherit from `SensorEntity` but `Entity` directly. But that is fixed now.